### PR TITLE
fix #79460 shop.autorenwelt.de

### LINKED ISSUE #79460: shop.autorenwelt.de
### Issue URL (Annoyance)
 [https://shop.autorenwelt.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fshop.autorenwelt.de%2F)
### Comment
Username: @andy-kl
> gray layer

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 beta 1
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2), <br>Disable cache for third-party requests, <br>Block Push API, <br>Block Location API, <br>Block Flash, <br>Block Java, <br>Hide your Referrer from third-parties (217.237.150.188)
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> Fanboy's Anti-Facebook List, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> Fanboy's Social Blocking List, <br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> Latvian List, <br/>Icelandic ABP List, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polish Anti Adblock Filters, <br/>Dandelion Sprout's Nordic Filters, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Spanish/Portuguese
Userscripts: | https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js
Referrer: | [217.237.150.188](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=217.237.150.188)

### PR DESCRIPTION
**Pull request by wout-codes**
#79460

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+shop.autorenwelt.de###cookie-banner_flag ~ #banner-wrapper
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:94]: `shop.autorenwelt.de###banner-wrapper` is enough

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -50,6 +50,7 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||api.nvidia.partners/telemetry/
 ||dax.yle.fi/yya/event
 ||nature.com/platform/track/
+||cdn.shopify.com/*/consent-tracking-api/*/consent-tracking-api.js
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:53]: It can break the store. Dangerous.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply SpywareFilter/sections/specific.txt:53]: Ok.

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt:53]: If I am not mistaken, it's in EasyPrivacy.

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt:53]: @al

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt:53]: @Alex-302

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:53]: not sure. Better remove, and block requests made by this script

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt:53]: I don't see anymore requests. I will remove it.

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt]: @ale

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt]: @Alex-302

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**